prprocessor/config/repos.yaml

```
@@ -74,6 +74,10 @@ theforeman/foreman_remote_execution:
   redmine: foreman_remote_execution
   redmine_required: true
   redmine_version_prefix: "foreman_remote_execution "
```

Unrelated, but it looks like the convention changed in the project and should now be:
```suggestion
  redmine_version_prefix: "foreman_remote_execution-"
```